src/formatDate.js

```diff
@@ -50,7 +50,68 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const resultTab = [];
+  const dateObj = {};
+
+  // save date using fromFormat-arg
```

Comment:
Something more like // transform and map date values to dateObj based on fromFormat